api/mod/tests/integration/conftest.py

```diff
@@ -0,0 +1,14 @@
+"""Fixtures for integration tests."""
+import pytest
+
+from mod.src.models.user import CreateUserRequest
+
+
+@pytest.fixture
+def user() -> CreateUserRequest:
+    """Creates a basic Annotation"""
+    return CreateUserRequest(
+        fullname="Prof. Dr. Folivora",
+        password="password",
+        email="team@folivora.online",
+    )
```

Comment:
If the database was seeded with `test_data.json`, we would only need this fixture for the users router and could move it there

Reply:
Will also be done in #86